Civi/Core/ClassScanner.php

```diff
@@ -142,7 +142,7 @@ private static function scanCoreClasses(): array {
     $classes = [];
     static::scanFolders($classes, $civicrmRoot, 'Civi/Test/ExampleData', '\\');
     // Most older CRM_ stuff doesn't implement event listeners & services so can be skipped.
-    static::scanFolders($classes, $civicrmRoot, 'CRM', '_', ';(Upgrade|Utils|Exception|_DAO|_Page|_Form|_Controller|_StateMachine|_Selector|_CodeGen);');
+    static::scanFolders($classes, $civicrmRoot, 'CRM', '_', ';(Upgrade|Utils|Exception|_DAO|_Page|_Controller|_StateMachine|_Selector|_CodeGen|_QuickForm);');
```

Comment:
@larssandergreen @seamuslee001  - I think this was good in defining the exclusion for `QuickForm`, but it should also retain the exclusion for `Form`. Update: #26937

Reply:
Oops, sorry about that.